src/Core/AspectKernel.php

```diff
@@ -216,9 +216,9 @@ protected function normalizeOptions(array $options)
         $options['excludePaths']   = PathResolver::realpath($options['excludePaths']);
 
         if (null === $options['annotationCache']) {
-            $options['annotationCache'] = new PhpFileCache(
+            $options['annotationCache'] = new FilesystemCache(
```

Comment:
This is extremely stupid and I don't have any idea why this causing issues...

If `Doctrine\Common\Cache\ PhpFileCache` is used, test fails (and it does not work in general) on PHP 5.x, and 7.0. It works on PHP 7.1 and 7.2.

On the other hand, with same settings, if `Doctrine\Common\Cache\ FilesystemCache` is used, all test passes and it does work.

@lisachenko -> do you have any idea what is happening here? And how to proceed? Should we keep `FilesystemCache` as default implementation, or? Version check and pick accordingly?

BTW I will report issue since it can be easily reproduced...